src/smvlang/expr2smv.cpp

```diff
@@ -656,6 +656,9 @@ expr2smvt::resultt expr2smvt::convert_rec(const exprt &src)
   else if(src.id() == ID_mod)
     return convert_binary(to_mod_expr(src), src.id_string(), precedencet::MULT);
 
+  else if(src.id() == ID_smv_set)
+    return convert_nondet_choice(src);
```

Comment:
This looks strange. Is this just a re-use of a function that wasn't meant to be used that way?

Reply:
Renamed the method